lib/util/git/__snapshots__/index.spec.ts.snap

```diff
@@ -1,7 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`platform/git getBranchCommit(branchName) should throw if branch does not exist 1`] = `[Error: Cannot fetch commit for branch that does not exist: not_found]`;
-
```

Comment:
As we don't throw anymore, can it lead to more requests like `GET /repos/some/repo/commits/undefined/statuses` and to increased `404` error rate in logs?

Reply:
I guess we should check each caller to see if it's relying on a throw